### PR TITLE
Added BLT Tomato (#2386)

### PR DESCRIPTION
**Pull request by nikhil25803**
## Screenshot
image

Please look into it @DonnieBLT.

### CODE REVIEW

**Review by DonnieBLT — DISMISSED** [review]
Thank you!

**Review by DonnieBLT — DISMISSED** [review]

File: website/templates/blt_tomato.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+            <!-- <ul>
+                {% for project in data %}
+                <li>
+                    <h2>{{ project.project_name }}</h2>
+                    <p>Repository: <a href="{{ project.repo_url }}" target="_blank">{{ project.repo_url }}</a></p>
+                    <p>Funding: {{ project.funding_hyperlinks|safe }}</p>
+                </li>
+                {% endfor %}
+            </ul> -->
```
Comment (arkid15r) [inline website/templates/blt_tomato.html:31]: Looks stale.

File: website/views.py [hunk 2]
```diff
@@ -4665,3 +4665,23 @@ def weekly_report(request):
         return HttpResponse("An error occurred while sending the weekly report")
 
     return HttpResponse("Weekly report sent successfully.")
+
+
+def blt_tomato(request):
+    current_dir = os.path.dirname(__file__)
+    json_file_path = os.path.join(current_dir, "fixtures", "blt_tomato_project_link.json")
```
Comment (arkid15r) [inline website/views.py:4672]: I suggest using `Path` instead.

File: website/views.py [hunk 3]
```diff
@@ -4665,3 +4665,23 @@ def weekly_report(request):
...
+        data = []
+
+    for project in data:
+        funding_details = project.get("funding_details", "").split(", ")
+        funding_links = [url.strip() for url in funding_details if url.startswith("https://")]
+
+        funding_link = funding_links[0] if funding_links else "#"
+        project["funding_hyperlinks"] = funding_link
+
+    return render(request, "blt_tomato.html", {"data": data})
```
Comment (arkid15r) [inline website/views.py:4687]:
```suggestion
    return render(request, "blt_tomato.html", {"projects": data})
```

File: website/templates/blt_tomato.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+    {% include "includes/sidenav.html" %}
+    <div class="min-h-screen flex flex-col mx-4 mt-5">
+        <h2 class="text-4xl font-semibold my-8 p-5 text-white rounded-md bg-[#d9534f]">BLT Tomato - This is an OWASP BLT project created to help other OWASP projects.</h2>
+        {% if data %}
```
Comment (arkid15r) [inline website/templates/blt_tomato.html:7]:
```suggestion
        {% if projects %}
```

File: website/templates/blt_tomato.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            <p class="text-xl font-semibold mt-5">The following OWASP projects are seeking funding and have a funding.yml file:</p>
+            <ul class="text-2xl">
+                {% for project in data %}
```
Comment (arkid15r) [inline website/templates/blt_tomato.html:10]:
```suggestion
                {% for project in projects %}
```

**Review by arkid15r — COMMENTED** [review]
Please take a look at my suggestions when you get a chance:

**Comment by nikhil25803** [comment]
Hey @arkid15r, I have made the changes accordingly. Please take a look once.

File: website/templates/blt_tomato.html [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+                            <a href="{{ project.funding_hyperlinks }}" target="_blank">
+                                <span class="text-white">Donate</span>
+                            </a>
+                        </button>                        
+                        <span class="ml-4">{{ project.funding_details }}</span>
+                    </li>
+                {% endfor %}
+            </ul>
+
+            <!-- <ul>
```
Comment (arkid15r) [inline website/templates/blt_tomato.html:23]: Could you remove the commented code?

**Review by arkid15r — COMMENTED** [review]
LGTM

File: website/templates/blt_tomato.html (hunk 6 above)
Comment (nikhil25803) [reply website/templates/blt_tomato.html:23]: Sure

**Review by DonnieBLT — DISMISSED** [review]

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**